app/_src/gateway-operator/guides/plugin-distribution.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Kong custom plugin distribution with KongPluginInstallation
+---
+
+
+
+{{ site.kgo_product_name }} can install Kong custom plugins packaged as container images. This guide shows how to package, install and use a custom plugin in {{site.base_gateway}} instances managed by the {{ site.kgo_product_name }}.
+
+{% include md/kgo/prerequisites.md version=page.version release=page.release kongplugininstallation=true %}
+
+{% include md/custom-plugin.md %}
+
+2. Build a container image that includes the plugin code.
+
+   It is expected to have plugin-related files at the root of the image. Thus for the plugin as mentioned earlier, the Dockerfile would look like this:
+
+   ```bash
+   echo 'FROM scratch
+
+   COPY myheader /
+   ' > Dockerfile
+   ```
+
+   where `myheader` is a directory that contains `handler.lua` and `schema.lua`.
+
+   Build the image:
+
+   ```bash
+   docker build -t myheader:1.0.0 .
+   ```
+
+   next push it to a registry available to the Kubernetes cluster where {{ site.kgo_product_name }} is running. A private registry can be used as well in the `KongPluginInstallation` resource.  Read more about it in CRD documentation.
```

Comment:
Can we link to the reference documentation? ( I believe this doesn't exist yet that's why we're not linking )